hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestCreateTable.scala

```diff
@@ -405,6 +405,145 @@ class TestCreateTable extends HoodieSparkSqlTestBase {
     }
   }
 
+  test("Test create table like") {
+    if (HoodieSparkUtils.gteqSpark3_1) {
+      // 1. Test create table from an existing HUDI table
+      withTempDir { tmp =>
```

Comment:
Spark2 will use Spark own `CreateTableLikeCommand`, we can't throw error here since we can't distinguish whether the user want to create hudi table or not.

```scala
* The syntax of using this command in SQL is(it doesn't support pass targetTable's provider):
 * {{{
 *   CREATE TABLE [IF NOT EXISTS] [db_name.]table_name
 *   LIKE [other_db_name.]existing_table_name [locationSpec]
 * }}}
```

Spark2 is becoming depreciated, maybe only supporting spark3+ is enough?

Reply:
We should avoid misusage of possible, or make it clear on the document.

Reply:
Updated doc here: https://github.com/apache/hudi/pull/9441